CompanyAddress.md

```diff
@@ -0,0 +1,130 @@
+Company Address Service
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Sample response](#sample-response)
+- [Versioning](#versioning)
+
+## Summary
+
+Company Address Service (CAS) takes a request containing a company name, latitude and longitude and returns a geocoded response.
```

Comment:
Is it useful to introduce the "CAS" acronym here? I would also start this sentence with the word "the".

Reply:
And maybe there's better phrasing than "a geocoded response"... maybe like, "a response describing a normalized geographic location"?